proc_MWE.py

```diff
@@ -1,27 +0,0 @@
-from pyspecdata import *
```

Comment:
this file is literally just plotting raw data and doing some waterfall function. I'm guessing MWE is just microwave enhancement but what specifically it is enhancing I do not know. This is probably one of the files originally pulled form the OLD proc_scripts when Alec first came in